sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -223,57 +261,27 @@ public final String getDescription() {
 
   /** Sampling decision without attributes. */
   @Immutable
-  private static final class SimpleDecision implements Decision {
-
-    private final boolean decision;
-
+  @AutoValue
+  abstract static class DecisionImpl implements Decision {
```

Comment:
javadoc on this class isn't correct any more.

Reply:
Done. Not necessary to have a javadoc for this class.